mount-tos-image/README.md

```diff
@@ -47,6 +50,12 @@ But since all calculations in the FAT are done using clusters all we need to do
 
 Of course this manipulation is not done directly on the SD-/CF-Card. Instead, the script makes heavy use of some nice and cool Linux features (FUSE and device mapper).
 
+## How to start the script
+
+* Connect the TOS medium of your choice as block device to the linux system or simply use an image file  
+* Start the script from within a working directory on a file system that is capable of creating spares files (e.g. btrfs, ext4, ...)
+* When started in "mount" mode, the script will create temporaray sparse files within the current directory 
```

Comment:
another typo here

Reply:
I found "temporaray" -> fixed that one, too!